hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/FilterParams.java

```diff
@@ -107,17 +126,50 @@ public String getFilterBySearchText() {
      *
      * @return the flag indicating if tagging filter is used
      */
-    public Boolean getSelectTargetWithNoTag() {
+    public boolean isSelectTargetWithNoTag() {
```

Comment:
revert

Reply:
why?

Reply:
This is the original peace of logic (at least the version from `TargetTable`):
`
    private long getTargetsCountWithFilter(final long totalTargetsCount, final Long pinnedDistId,
            final FilterParams filterParams) {
        final Optional<Long> query = managementUIState.getTargetTableFilters().getTargetFilterQuery();

        final long size;
        if (query.isPresent()) {
            size = targetManagement.countByTargetFilterQuery(query.get());
        } else if (noFilterSelected(filterParams.getFilterByStatus(), pinnedDistId,
                filterParams.getSelectTargetWithNoTag(), filterParams.getFilterByTagNames(),
                filterParams.getFilterBySearchText())) {
            size = totalTargetsCount;
        } else {
            size = targetManagement.countByFilters(filterParams.getFilterByStatus(), filterParams.getOverdueState(),
                    filterParams.getFilterBySearchText(), filterParams.getFilterByDistributionId(),
                    filterParams.getSelectTargetWithNoTag(), filterParams.getFilterByTagNames());
        }
        return size;
    }

    private static boolean noFilterSelected(final Collection<TargetUpdateStatus> status, final Long distributionId,
            final Boolean noTagClicked, final String[] targetTags, final String searchText) {
        return CollectionUtils.isEmpty(status) && distributionId == null && StringUtils.isEmpty(searchText)
                && !isTagSelected(targetTags, noTagClicked);
    }

    private static Boolean isTagSelected(final String[] targetTags, final Boolean noTagClicked) {
        return targetTags == null && !noTagClicked;
    }
`